gui/src/components/logs/FormattedLogMessage.svelte

```diff
@@ -1,20 +1,51 @@
 <script lang="ts">
-  import { parseSpans } from './parseSpans';
+  import { parseSpans, Span } from './parseSpans';
 
   export let message: string = '';
 
   const spans = parseSpans(message);
+
+  const spanStyle = (span: Span): string => {
+    const styles: string[] = [];
+    const addStyle = (key: string, value: string) => {
+      styles.push(`${key}: ${value}`);
+    };
+    if (span.forground != null) {
+      addStyle('color', span.forground);
```

Comment:
Nit: "foreground" instead of "forground"

Reply:
d'oh - will fix